client/login/wp-login/index.jsx

```diff
@@ -70,10 +70,11 @@ export class Login extends React.Component {
 	}
 
 	render() {
-		const { translate, twoFactorAuthType } = this.props;
+		const { translate, twoFactorAuthType, clientId } = this.props;
+		const title = clientId ? translate( 'Howdy! Log in to ' + clientId + ' with your WordPress.com account.' ) : translate( 'Log in to your account.' );
```

Comment:
That works for me, but it is a change in implementation so we'd need to talk to someone else about it probably, particularly Woo (cc @garymurray)